client/signup/controller.js

```diff
@@ -69,6 +71,10 @@ export default {
 	},
 
 	redirectToFlow( context, next ) {
+		if ( utils.getFlowName( context.params ) === defaultFlowName && abtest( 'freeTrials' ) === 'offered' ) {
+			return page.redirect( '/start/free-trial' );
```

Comment:
Also minor, but we should use a function from `client/signup/utils` to get this URL. Probably

```js
getValidPath( {
  flowName: 'free-trial'
} );
```